ait/commons/util/command/upload.py

```diff
@@ -27,7 +78,7 @@ def run(self):
 
         try:
 
-            # filter out any duplicate path after expansion 
+            # filter out any duplicate path after expansion
```

Comment:
Who is this comment for?
If it need to be implement, do it, otherwise delete it.

Reply:
no idea, this was actually part of the code base from before. I'll remove it